.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,19 +19,18 @@ These points need to be fulfilled for every PR.
   - [ ] reformat the code with `scripts/dev/reformat-all`
   - [ ] make all unit tests pass
   - [ ] fix all memleaks
+  - [ ] fix the CI itself (or rebase if already fixed)
 - [ ] The PR is rebased with current master.
 
 <!--
-If you have any troubles fulfilling these criteria, please write
-about the trouble as comment in the PR. We will help you,
-but we cannot accept PRs that do not fulfill the basics.
+If you have any troubles fulfilling these criteria, please write about the trouble as comment in the PR.
+We will help you, but we cannot accept PRs that do not fulfill the basics.
 -->
 
 ## Checklist
 
 <!--
-For docu fixes, spell checking, and similar none of these points below
-need to be checked.
+For docu fixes, spell checking, and similar none of these points below need to be checked.
```

Comment:
```suggestion
For documentation fixes, spell checking, and similar none of these points below need to be checked.
```

I personally have never encountered anyone saying "docu" in English.

Reply:
Yes, in English it is normally shortened to "docs".